terraform/variables.tf

```diff
@@ -173,3 +179,26 @@ variable "distributed_tff_example_worker_2_address" {
   description = "Address of the second worker of the distributed TensorFlow Federated example. Set this when the worker is outside the coordinator mesh."
   type        = string
 }
+
+variable "cross-device" {
+  description = "Enable cross device infrastructure deployment"
+  type        = bool
+  default     = false
+}
+
+variable "spanner_instance_config" {
+  description = "Multi region config value for the Spanner Instance. Example: 'nam10' for North America."
+  type        = string
+  default     = "regional-europe-west1"
+}
+
+variable "spanner_processing_units" {
+  description = "Spanner's compute capacity. 1000 processing units = 1 node and must be set as a multiple of 100."
+  type        = number
+  default     = 1000
+}
+
+variable "tenant_namespace" {
```

Comment:
Rename this variable to something more specific. `cross_device_workloads_kubernetes_namespace` or something similar?